StarRailBattleSim/src/relics/relics/GuardInTheWutheringSnow.java

```diff
@@ -0,0 +1,30 @@
+package relics.relics;
+
+import characters.AbstractCharacter;
+import powers.PermPower;
+import powers.PowerStat;
+import relics.AbstractRelicSetBonus;
+
+public class GuardInTheWutheringSnow extends AbstractRelicSetBonus {
+    public GuardInTheWutheringSnow(AbstractCharacter owner) {
+        super(owner);
+    }
+
+    public GuardInTheWutheringSnow(AbstractCharacter owner, boolean fullSet) {
+        super(owner, fullSet);
+    }
+
+    @Override
+    public void onEquip() {
+        this.owner.addPower(PermPower.create(PowerStat.DEFENSE_REDUCTION, 8, "Guard in the Wuthering Snow DMG reduction"));
```

Comment:
right now this is just reducing the wearer's defense by 8% which would result in taking more damage 😝 (not that damage dealt to the characters is actually being simulated . . .)

Reply:
whoops... 😂